### PR TITLE
added  comments to my project file

### PR DESCRIPTION
**Pull request by smitM76**

### CODE REVIEW

**Review by codingquark — CHANGES_REQUESTED** [review]
Comments are not supposed to say what code is, but what code is for.

It should complement your code. Help others understand the reasoning behind the code.

File: RestXam2_O/App17/App17/App17/MainPage.xaml.cs [hunk 1]
```diff
@@ -13,7 +13,12 @@
 
 namespace App17
 {
-    //creating classes for binding response data Meta,Object,RootObject
+    /*
+    creating classes data Meta,Object,RootObject with getter setter method
+    this classes veries as per json fields
```
Comment (codingquark) [inline RestXam2_O/App17/App17/App17/MainPage.xaml.cs]: Change `veries` to `varies`

File: RestXam2_O/App17/App17/App17/MainPage.xaml.cs [hunk 2]
```diff
@@ -13,7 +13,12 @@
 
 namespace App17
 {
-    //creating classes for binding response data Meta,Object,RootObject
+    /*
+    creating classes data Meta,Object,RootObject with getter setter method
+    this classes veries as per json fields 
+    used "http://json2csharp.com/" to auto generate this classes
+    which helps us to bind and hendles the json responce we gonna get from rpc call
```
Comment (codingquark) [inline RestXam2_O/App17/App17/App17/MainPage.xaml.cs]: `hendles` to `handles`

Comment (codingquark) [reply RestXam2_O/App17/App17/App17/MainPage.xaml.cs]: `responce` to `response`

**Review by codingquark — CHANGES_REQUESTED** [review]

File: RestXam2_O/App17/App17/App17/MainPage.xaml.cs (hunk 2 above)
Comment (smitM76) [reply RestXam2_O/App17/App17/App17/MainPage.xaml.cs]: okk

**Review by codingquark — APPROVED** [review]

**Pull request merged**